content/influxdb/cloud-dedicated/primers/api/v1/_index.md

```diff
@@ -0,0 +1,275 @@
+
+{{% code-tab-content %}}
+
+With an empty username:
+
+```js
+{{% get-shared-text "api/cloud-dedicated/querystring-auth.js" %}}
+```
+{{% /code-tab-content %}}
+{{< /code-tabs-wrapper >}}
+
+Replace the following:
+
+- **`DATABASE_TOKEN`**: a [Database Token](/influxdb/cloud-dedicated/get-started/setup/#create-a-database-token) with sufficient permissions to the database.
+
+### Authenticate with the Token scheme
+
+Use the `Authorization` header with the `Token` scheme to authenticate `/write` and `/query` requests.
+
+#### Syntax
+
+```http
+Authorization: Token DATABASE_TOKEN
+```
+
+#### Example
+
+{{< code-tabs-wrapper >}}
+{{% code-tabs %}}
+[curl](#curl)
+[Node.js](#nodejs)
+{{% /code-tabs %}}
+{{% code-tab-content %}}
+```sh
+{{% get-shared-text "api/cloud-dedicated/token-auth.sh" %}}
+```
+{{% /code-tab-content %}}
+{{% code-tab-content %}}
+```js
+{{% get-shared-text "api/cloud-dedicated/token-auth.js" %}}
+```
+{{% /code-tab-content %}}
+{{< /code-tabs-wrapper >}}
+
+<!-- ## Responses -->
+
+## Write data with the v1 API
+
+### Write data using Telegraf
+
+If have existing v1 workloads that use Telegraf,
+you can use the [InfluxDB v1.x `outputs.influxdb` plugin](https://github.com/influxdata/telegraf/blob/release-1.26/plugins/outputs/influxdb/README.md) to write data.
+In order to write data to InfluxDB Cloud Dedicated, you'll need to
+make the following changes to your `outputs.influxdb` configuration:
+
+Parameter          | Ignored                  | Value
+-------------------|--------------------------|------------------------------------------------------------
+`database`         | Honored                  | Database name
+`retention_policy` | Honored, but discouraged | [Duration](/influxdb/cloud-iox/reference/glossary/#duration)
+`username`         | Ignored                  | Any string or empty
+`password`         | Honored                  | [Database token](/influxdb/cloud-dedicated/get-started/setup/#create-a-database-token) with permission to write to the database
+`content_encoding`       | Honored                    | `gzip` (compressed) or `identity` (uncompressed)
+`skip_database_creation` | Ignored                    | N/A ([contact Support]() to create a database)
+
+The following sample shows how to configure the `outputs.influxdb` Telegraf plugin for InfluxDB Cloud Dedicated:
+
+```toml
+[[outputs.influxdb]]
+  urls = ["http://localhost:8086"]
+  database = "DATABASE_NAME"
+  skip_database_creation = true
+  retention_policy = ""
+  username = ""
+  password = "DATABASE_TOKEN"
+  content_encoding = "gzip”
+```
+
+Replace the following:
+
+- **`DATABASE_NAME`**: your InfluxDB Cloud Dedicated database
+- **`DATABASE_TOKEN`**: a [Database Token](/influxdb/cloud-dedicated/get-started/setup/#create-a-database-token) with permission to write to the database
+
+#### Other Telegraf configuration options
+
+`influx_uint_support`: supported in InfluxDB IOx.
+
+### Write data using v1 client libraries
+
```

Comment:
Still need content here. Client libraries should still work, they just need to update the host and password credentials.

Reply:
Opened issue #4869